app/views/bookings/show.html.erb

```diff
@@ -0,0 +1,10 @@
+<h1>Congratulations!</h1>
+<h1>You've booked with <%= @booking.cooker.first_name.capitalize %> <%= @booking.cooker.last_name.capitalize %>!</h1>
```

Comment:
@Lulu92270 here in the gif you shared on Slack, it displays the name "D O", not the name of marion, but I think you want to display the name of the cook, right??
Because I think "D" looks like it's you.